client/lib/abtest/active-tests.js

```diff
@@ -124,6 +124,13 @@ export default {
 		defaultVariation: 'control',
 		allowExistingUsers: true,
 	},
+	checklistUpdateHomepage: {
+		datestamp: '20191108',
+		variations: {
+			templateSelector: 0,
+			control: 100,
+    }
```

Comment:
This needs a default variation to work, otherwise we get a WSOD when opening `/home`.
```js
checklistUpdateHomepage: {
		datestamp: '20191108',
		variations: {
			templateSelector: 0,
			control: 100,
    },
    defaultVariation: 'control',
}
```

Reply:
Thanks!  I think I lost this in a rebase.  Adding back in!